.buildkite/filebeat/filebeat-pipeline.yml

```diff
@@ -317,6 +318,38 @@ steps:
           - github_commit_status:
               context: "filebeat: Win 2022 Unit Tests"
 
+      - label: ":ubuntu: Filebeat: FIPS ECH Integration Tests"
+        env:
+          ASDF_TERRAFORM_VERSION: "1.9.3"
+          ASDF_PYTHON_VERSION: "3.9.13" # Not needed by ECH tests, but needed by VM
+        command: |
+          .buildkite/scripts/custom_fips_ech_test.sh filebeat
+        retry:
+          automatic:
+            - limit: 1
+        agents:
+          provider: "aws"
+          imagePrefix: "${IMAGE_UBUNTU_X86_64_FIPS}"
+          instanceType: "m5.xlarge"
+        artifact_paths:
+          - "filebeat/build/*.xml"
+          - "filebeat/build/*.json"
+          - "filebeat/build/integration-tests/*"
+          - "filebeat/build/integration-tests/Test*/*"
+          - "filebeat/build/integration-tests/Test*/data/**/*"
+        plugins:
+          - test-collector#v1.10.2:
+              files: "filebeat/build/TEST-*.xml"
+              format: "junit"
+              branches: "main"
+              debug: true
+          - elastic/oblt-cli#v0.2.0:
```

Comment:
You also need access to the `google secrets` and configure the OIDC:
- use https://github.com/elastic/oblt-google-auth-buildkite-plugin when accessing the secrets using `oblt-cli cluster secrets env`
- then ping me and I'll configure the OIDC trust for the BK pipelines in `elastic/beats` that need access to it